bin/cognito-idp.ts

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from '@aws-cdk/core';
+import { CognitoIdpStack, CognitoIdpStackProps } from '../lib/cognito-idp-stack';
+import * as fs from 'fs-extra';
+
+
+const app = new cdk.App();
+
+// For local development, create a copy of config/env.json called config/env-local.json
+const localConfigPath = './config/env-local.json';
+if (fs.existsSync(localConfigPath)) {
+
+    // Deploy to your 'local' development account
+
+    // tslint:disable-next-line: no-var-requires
+    const localStack = new CognitoIdpStack(app, 'CognitoIdp-Local', require('.' + localConfigPath));
+}
+
+// tslint:disable-next-line: no-var-requires
+const prodStack = new CognitoIdpStack(app, 'CognitoIdp-Prod', require('../config/env-prod.json'));
```

Comment:
Now that there is no hard synth requirements, do we still need this fork and `env-prod.json`?

Reply:
Yes, because we don't want the local stack to be attempted on a build server that doesn't have the local env settings. It's more obvious in a project with a pipeline, and it would also cause a problem if a user ends up adding a lookup to the code.